sklearn/utils/estimator_checks.py

```diff
@@ -1360,6 +1390,62 @@ def check_classifiers_train(name, classifier_orig):
                 assert_array_equal(np.argsort(y_log_prob), np.argsort(y_prob))
 
 
+def check_outliers_train(name, estimator_orig):
```

Comment:
this check does more than train as it does some predict. Is it just a bad naming?

Reply:
forget it. It's just consistent with classifier test.